vspace/src/main/java/edu/asu/diging/vspace/web/staff/ExhibitionConfigurationController.java

```diff
@@ -99,16 +109,123 @@ public RedirectView createOrUpdateExhibition(HttpServletRequest request,
         exhibition.setStartSpace(startSpace);
         exhibition.setTitle(title);
         exhibition.setMode(exhibitMode);
-        exhibitManager.updateExhibitionLanguages(exhibition,languages,defaultLanguage);
-    
-        if(exhibitMode.equals(ExhibitionModes.OFFLINE) && !customMessage.equals(ExhibitionModes.OFFLINE.getValue())) {
+
+        exhibitManager.updateExhibitionLanguages(exhibition, languages, defaultLanguage);
+
+        if (exhibitMode.equals(ExhibitionModes.OFFLINE) && !customMessage.equals(ExhibitionModes.OFFLINE.getValue())) {
 
             exhibition.setCustomMessage(customMessage);
         }
+
+        exhibition = (Exhibition) exhibitManager.storeExhibition(exhibition);
+        attributes.addAllAttributes(addSuccessAttributes(exhibition));
+
+        return new RedirectView(request.getContextPath() + "/staff/exhibit/config");
+    }
+    
+    /**
+     * To create or update the default space link image.
+     * 
+     * @param exhibitID - the ID of the exhibition to update
+     * @param spaceLinkImage - the MultipartFile representing the space link image to be uploaded
+     * @param attributes - the RedirectAttributes object to add flash attributes
+     * @return a RedirectView to the exhibition configuration page
+     * @throws IOException if an input or output error occurs
+     */
+    @RequestMapping(value = "/staff/exhibit/config/spaceLinkImage", method = RequestMethod.POST)
+    public RedirectView createOrUpdateSpaceLinkImage(HttpServletRequest request,
```

Comment:
these methods should go in their own controller

Reply:
I have removed the 3 different methods used and changed the name of this method, so kept it as it is in the same controller because it is a common method for default link images